host.defs

```diff
@@ -527,7 +527,7 @@ enddef	:
 newdef	: defined(NetBSD) 
 comment : NetBSD
 vendor	: defined(aarch64)				: "arm"
-vendor	: defined(algor)				: "algoritmics"
+vendor	: defined(algor)				: "algorithmics"
```

Comment:
This refers to ALGOR, Inc. Do you have some reference that the longer form should have an "h" in there?